api/features/signals.py

```diff
@@ -12,6 +18,34 @@
 
 @receiver(post_save, sender=FeatureState)
 def trigger_feature_state_change_webhooks_signal(instance, **kwargs):
+    # Enqueue Github integration tasks if feature has GitHub external resources
+    if (
+        not instance.identity_id
+        and not instance.feature_segment
+        and instance.feature.external_resources.exists()
+        and instance.environment.project.github_project.exists()
+        and hasattr(instance.environment.project.organisation, "github_config")
```

Comment:
I'm not sure `hasattr` is used correctly here.

Also I'd love to see a comment explaining the lengthy condition.

Reply:
```suggestion
        and instance.environment.project.organisation
```

or, to be more explicit

```suggestion
        and instance.environment.project.organisation is not None
```

Reply:
This doesn't work in this case, using exists() instead.